src/sql/azdata.proposed.d.ts

```diff
@@ -310,6 +316,7 @@ declare module 'azdata' {
 		export interface ModelViewDashboard {
 			registerTabs(handler: (view: ModelView) => Thenable<(DashboardTab | DashboardTabGroup)[]>): void;
 			open(): Thenable<void>;
+			updateTabs(tabs: (DashboardTab | DashboardTabGroup)[]): void;
```

Comment:
It seems weird to have both registerTabs and updateTabs - can we just get rid of registerTabs?

Reply:
registerTabs is for initializing the dashboard, which will call registerContent behind the scene. it is to tell the model-view framework to initialize the container control, I don't think this can be removed.